src/pages/Login.vue

```diff
@@ -0,0 +1,108 @@
+<script setup>
+    import { reactive } from 'vue';
+    import router from '../routers/Router'
```

Comment:
muốn sử dụng router ở trong app thì dùng useRouter() composable em nhé
=> tìm hiểu useRouter của vue-router nhé
```suggestion
    import { useRouter } from 'vue-router'
    
    const router = useRouter()
```